oh_staff_ui/classes/ImageFileHandler.py

```diff
@@ -127,3 +129,17 @@ def _get_new_image_dimensions(
         scale_factor = max(current_sizes) / max_size
         new_sizes = tuple(int(dimension / scale_factor) for dimension in current_sizes)
         return new_sizes
+
+    def _make_square_thumbnail(self, thumbnail_image: Image.Image) -> Image.Image:
```

Comment:
Please add a test for the thumbnail size to `MediaFileTestCase`, to confirm the size is 200x200 (more appropriately, `settings.IMAGE_SETTINGS["thumbnail_long_dimension"]` square).